client/src/AppBundle/Command/DocumentSyncCommand.php

```diff
@@ -69,7 +69,38 @@ private function getQueuedDocuments(): array
     {
         $options = [
             'query' => [
-                'groups' => ['documents', 'document-synchronisation', 'document-storage-reference', 'document-report', 'report', 'report-client', 'client-case-number']
+                'groups' => [
+                    'documents',
+                    'document-synchronisation',
+                    'document-storage-reference',
+                    'document-report',
+                    'report',
+                    'report-client',
+                    'client-case-number',
+                    'document-report-submission',
+                    'report-submission',
+                    'documentSync',
```

Comment:
With the new nested groups, I think there are some duplicates in here that can be removed?
```suggestion
                    'document-report-submission',
```

Reply:
Just had a quick test and I agree with all of these apart from `document-report-submission` and `report-submission` as these bring back required properties on Document.